editoast/src/models/rolling_stock/mod.rs

```diff
@@ -129,6 +132,35 @@ pub struct RollingStockModel {
     pub version: Option<i64>,
 }
 
+fn validate_rolling_stock(
+    rolling_stock: &RollingStockModel,
+) -> std::result::Result<(), ValidationError> {
+    let electric = rolling_stock.is_electric();
+    if electric {
```

Comment:
One level of indentation can easily be removed.

```suggestion
    if !rolling_stock.is_electric() {
        return Ok(());
    }
```